server/protocols/jmap-draft-integration-testing/cassandra-jmap-draft-integration-testing/src/test/java/org/apache/james/jmap/cassandra/CassandraGetMessageListMethodTest.java

```diff
@@ -46,4 +55,29 @@ protected GuiceJamesServer createJmapServer() throws IOException {
     protected void await() {
         rule.await();
     }
+
+    @Ignore("Demonstrate James respond 400 upon unavailable ElasticSearch")
```

Comment:
s/400/500
in commit names too

Reply:
No right now we respond 400, not 503 (which is the expected result)